app/views/content/welcome/jacks-story/_article.html.erb

```diff
@@ -1,4 +1,4 @@
-<article class="story">
+<article class="case-study">
   <p>
     Jack is a PE teacher from Leeds, with a passion for opening up conversations
```

Comment:
Should we make this intro (which is in the third person) look different from the rest of the article, which is in the first person? Or put quotes round the first person section?